File: Day 3/leapYear.py
# 🚨 Don't change the code below 👇
year = int(input("Which year do you want to check? "))
# 🚨 Don't change the code above 👆

#Write your code below this line 👇

if year % 4 == 0:
    if year % 100 == 0 and year % 400 == 0:
        print(f"{year} is a leap year")
    elif year % 100 == 0:
        print(f"{year} is not a leap year")
    else:
        print(f"{year} is a leap year")    
else:
    print(f"{year} is not a leap year") 
        

File: Day 3/leapYear.js

let leap = x => {
    if (x % 4 === 0) {
        if (x % 100 === 0 && x % 400 === 0) {
            console.log("Leap year 1");
        } else if(x % 100 === 0) {
            console.log("Not leap year 1")
        } else {
            console.log("leap year 2")
        }
    } else {
        console.log("Not a leap year 2")
    }
}